example/feature_demo/demo_types.proto

```diff
@@ -70,7 +70,8 @@ message TypeWithID {
   option (gorm.opts) = {
     ormable: true,
     include: [
-      {type: "int32", name: "secret_int", tag: {ignore: true}}
+      {type: "int32", name: "secret_int", tag: {ignore: true}},
+      {type: "[]*JoinTable", name: "multi_account_types", tag: {foreign_key: "TypeWithIDID"}}
```

Comment:
This is my particular use case for this PR: being able to specify a GORM model that shouldn't be visible to the API spec, but still available to the GORM models